crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -945,12 +947,23 @@ def upgrade_first_cluster_node(node, other_node)
         other_node_api.disable_pre_upgrade_attribute_for node.name
         # delete old pacemaker resources (from the node where old pacemaker is running)
         delete_pacemaker_resources other_node
+        # set stateless clones to false if needed after deleting pacemaker resources
+        enable_clone_stateless_services(node)
         # start crowbar-join at the first node
         node_api.post_upgrade
         node_api.join_and_chef
         node_api.save_node_state("controller", "upgraded")
       end
 
+      def enable_clone_stateless_services(node)
+        pacemaker_role_name = node.roles.grep(/^pacemaker-config-.*/).first
+        return if pacemaker_role_name.nil?
+        pacemaker_role = RoleObject.find_role_by_name(pacemaker_role_name)
+        return if pacemaker_role.default_attributes["pacemaker"]["clone_stateless_services"]
+        pacemaker_role.default_attributes["pacemaker"]["clone_stateless_services"] = false
```

Comment:
The method name says enable, but this sets it to false?

Reply:
Lol, turns out I need to refresh the page before I press 'add comment'. As I was called out of my office for about an hour between writing a comment and pressing save :P

Reply:
Indeed the name is wrong. I changed it quickly as hound complained about it being named `set_*` but the current name has no sense.